doc/python/renderers.md

```diff
@@ -33,19 +33,21 @@ jupyter:
     thumbnail: thumbnail/displaying-figures.png
     order: 1
 ---
+# Displaying Figures 
 
-### Displaying plotly figures
-This section covers the many ways to display plotly figures from Python.  At the highest level, there are three general approaches:
+Plotly's Python graphing library, `plotly.py`, gives you a wide range of options for how and where to display your figures. 
 
- 1. Using the renderers framework in the context of a script or notebook
- 2. Using Dash in a web app context
- 3. Using a `FigureWidget` in an ipywidgets context
+In general, there are three different approaches you can take in order to display figures:
+
+ 1. Using the `renderers` framework in the context of a script or notebook
```

Comment:
Here also I think the usage of backticks for `renderers` is going a bit too far IMO...